Behavioral/State/ContextOrder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DesignPatterns\Behavioral\State;
+
+class ContextOrder extends StateOrder
+{
+    public function getState():StateOrder
```

Comment:
What you say about https://github.com/domnikl/DesignPatternsPHP/blob/master/Structural/Adapter/Book.php#L22?